source/jormungandr/jormungandr/fallback_modes.py

```diff
@@ -89,3 +89,12 @@ def get_allowed_combinations_str(cls):
 all_fallback_modes = FallbackModes.modes_str()
 
 allowed_combinations = FallbackModes.get_allowed_combinations_str()
+
+street_network_modes_in_db = {
```

Comment:
To express string enumeration from a db, you could use [sqlalchemy.types.Enum](https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.Enum) or a simple [class enum.Enum](https://docs.python.org/3/library/enum.html#module-enum) could do the trick.